Fixed error sql in deployment.xml [insertDeployment and bulkInsertDeployment]

ScathonLin opened the pull request.

ScathonLin committed:
[issue]Fixed Sql Error Of insertDeployment And buldInsertDeployment in Deployment.xml
[author] linhuadong(ScathonLin)
[date] 2020-04-16 22:06
[github]https://github.com/ScathonLin

ScathonLin committed:
[issue]Fixed Sql Error Of insertDeployment And buldInsertDeployment in Deployment.xml
[author] linhuadong(ScathonLin)
[date] 2020-04-16 22:06
[github]https://github.com/ScathonLin

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Activiti/Activiti?pullRequest=3233) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Activiti/Activiti?pullRequest=3233) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: mteodori<br/>:x: ScathonLin<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Activiti/Activiti?pullRequest=3233) it.</sub>

ScathonLin committed:
[issue]Fixed Sql Error Of insertDeployment And buldInsertDeployment in Deployment.xml
[author] linhuadong(ScathonLin)
[date] 2020-04-16 22:06
[github]https://github.com/ScathonLin

ScathonLin committed:
Merge branch 'develop' into develop

mergify[bot] committed:
Merge branch 'develop' into develop

mteodori committed:
Merge branch 'develop' into develop

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Activiti/Activiti?pullRequest=3233) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Activiti/Activiti?pullRequest=3233) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: ScathonLin<br/>:x: mteodori<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Activiti/Activiti?pullRequest=3233) it.</sub>
